doc/api/conf.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../../src'))
+# For soupparser in lxml.html.ElementSoup module:
+sys.path.insert(1, os.path.abspath('../../src/lxml/html'))
```

Comment:
That's a bug in the `ElementSoup` (legacy) module. Suggests that no-one has been using that essentially forever.
```suggestion
```